src/SWP/SuperdeskBridgeBundle/Bridge/Bridge.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+/**
+ * @copyright 2015 Sourcefabric z.ú.
+ * @author Mischa Gorinskat <mischa.gorinskat@sourcefabric.org>
+ * @license http://www.gnu.org/licenses/gpl-3.0.txt
+ */
+
+namespace SWP\SuperdeskBridgeBundle\Bridge;
+
+use SWP\SuperdeskBridgeBundle\Client\Client;
+use SWP\SuperdeskBridgeBundle\Data\Item;
+use SWP\SuperdeskBridgeBundle\Data\Package;
+
+/**
+ * Superdesk Bridge class.
+ */
+class Bridge
```

Comment:
i would create sperate component called superdesk/content-api-php-sdk and keep all mentioned by be in comments classes there and use them in bundle from that component required by composer